internal/cmd/keys.go

```diff
@@ -112,28 +110,20 @@ func newKeysRemoveCmd() *cobra.Command {
 				return err
 			}
 
-			if keys.Count == 0 {
-				return fmt.Errorf("no access key found with this name")
-			}
-
-			if keys.Count != 1 {
-				return fmt.Errorf("invalid access key response, there should only be one access key that matches a name, but multiple were found")
-			}
-
-			key := keys.Items[0]
+			for _, key := range keys.Items {
+				err = client.DeleteAccessKey(key.ID)
+				if err != nil {
+					return err
+				}
 
-			err = client.DeleteAccessKey(key.ID)
-			if err != nil {
-				return err
-			}
+				issuedFor := key.IssuedForName
```

Comment:
What is this check for?

Reply:
This is to set `issuedFor` in case `issuedForName` is empty. I believe it's for compatibility between the CLI and older version of the API which does not set `issuedForName`